insights/client/connection.py

```diff
@@ -245,7 +245,7 @@ def _test_urls(self, url, method):
                 else:
                     logger.info("Connection failed")
                     return False
-            except requests.ConnectionError as exc:
+            except (requests.Timeout, requests.ConnectionError) as exc:
```

Comment:
Does this list-like syntax catch the exception if it's either a `requests.Timeout` or a `requests.ConnectionError`?